packages/sheets-formula-ui/src/controllers/formula-clipboard.controller.ts

```diff
@@ -386,83 +386,155 @@ function getSpecialPasteFormulaValueMatrix(
     return valueMatrix;
 }
 
+// eslint-disable-next-line
 function getDefaultPasteValueMatrix(
```

Comment:
Should have this unit test covered.

Reply:
It is a bette approach to break this function into several smaller and more maintainable pieces instead of disable ESLine warning.